front-end/src/LoginButton.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+
+
+function LoginButton() {
+  return (
+	<Link to={'/Login'}>
```

Comment:
We actually may have to use `a href` here--or remove the interior button element entirely? Right now they don't redirect to the login or signup page.